### PR TITLE
Auto OCP-19196 and OCP-16495

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe please help review, thx
Depend on https://github.com/openshift/verification-tests/pull/566

### CODE REVIEW

File: features/registry/is.feature [hunk 1]
```diff
@@ -86,3 +86,87 @@ Feature: Testing imagestream
...
+  # @case_id OCP-16495
+  @destructive
+  @admin
+  Scenario: Do not prune layer of a valid Image due to minimum aging
+    Given I have a project
+    And evaluation of `project.name` is stored in the :u1p1 clipboard
+    Given I enable image-registry default route
+    Given default image registry route is stored in the :registry_hostname clipboard
+    Given certification for default image registry is saved
+
```
Comment (wewang58) [inline features/registry/is.feature]: There are some blank lines, is ti intended?

Comment (xiuwang) [reply features/registry/is.feature]: It's intended

**Comment by liangxia** [comment]
Could you add logs for admin and destructive test cases so we know better what kind of change is made to the cluster ?

**Comment by xiuwang** [comment]
> Could you add logs for admin and destructive test cases so we know better what kind of change is made to the cluster ?
@liangxia This pr is related to #566，it's not merged yet
So I used another admin &destructive cases log paste here https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/60494/consoleFull

**Comment by xiuwang** [comment]
@pruan-rht @akostadinov Could you help review this again or merge it?
Thanks

**Pull request merged**